tests/plugins/identity/user.py

```diff
@@ -0,0 +1,11 @@
+from server.apps.identity.models import User
+from mixer.backend.django import mixer
+
+import pytest
+
+
+@pytest.fixture
+def user() -> User:
```

Comment:
nitpick: Wouldn't it be more accurate to use the `Generator` type annotation? Or this is common practice?

https://docs.python.org/3.11/library/typing.html#typing.Generator
```suggestion
def user() -> Iterator[User]:
```